demo/molecule/field/demo/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "demo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@s-ui/react-atom-spinner": "1",
+    "@s-ui/react-atom-textarea": "1",
+    "@s-ui/react-form-input": "1",
+    "@s-ui/react-molecule-field-textarea": "1",
+    "formik": "^1.0.2",
```

Comment:
use exact version please, you can npm install with -DE (save dev exact) or -SE (save exact) in order do not add the ^